seata-spring-boot-starter/src/main/java/io/seata/spring/boot/autoconfigure/properties/file/SpringProperties.java

```diff
@@ -0,0 +1,44 @@
+/*
+ *  Copyright 1999-2019 Seata.io Group.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+package io.seata.spring.boot.autoconfigure.properties.file;
+
+import org.springframework.boot.autoconfigure.condition.ConditionalOnProperty;
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.stereotype.Component;
+
+import static io.seata.spring.boot.autoconfigure.StarterConstants.SPRING_PREFIX;
+
+/**
+ * @author xingfudeshi@gmail.com
+ * @date 2019/09/30
+ */
+@Component
+@ConfigurationProperties(prefix = SPRING_PREFIX)
+public class SpringProperties {
+    /**
+     * auto proxy the DataSource bean
+     */
+    private boolean datasourceAutoproxy = true;
```

Comment:
![image](https://user-images.githubusercontent.com/26616196/66620434-08578d00-ec13-11e9-8046-fc52a8da6f56.png)
the datasourceAutoproxy default is false?

Reply:
When using seata-spring-boot-starter,the default value will be true,Without using seata-spring-boot-starter,the dfault value is false.